opensrp-app/src/test/java/org/smartregister/EnvironmentManagerTest.java

```diff
@@ -0,0 +1,82 @@
+package org.smartregister;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Answers;
+import org.mockito.Mock;
+import org.robolectric.util.ReflectionHelpers;
+import org.smartregister.domain.Environment;
+import org.smartregister.util.JsonFormUtils;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+
+/**
+ * Created by Ephraim Kigamba - nek.eam@gmail.com on 27-04-2021.
+ */
+public class EnvironmentManagerTest extends BaseRobolectricUnitTest {
+
+    @Mock(answer = Answers.CALLS_REAL_METHODS)
+    private EnvironmentManager environmentManager;
+
+    @Test
+    public void getEnvironments() {
```

Comment:
The method name could be a little bit more descriptive to understand what's being tested

Reply:
The test just tests that all the environments are returned. It is a getter. We do not have a variation or special case. Loading the environments is tested on a different test and functionality lives elsewhere. The method only has one simple line https://github.com/opensrp/opensrp-client-core/blob/master/opensrp-app/src/main/java/org/smartregister/EnvironmentManager.java#L36